src/SSHDebugPS/Docker/DockerHelper.cs

```diff
@@ -17,6 +17,123 @@ public class DockerHelper
         private const string dockerPSCommand = "ps";
         // --no-trunc avoids parameter truncation
         private const string dockerPSArgs = "-f status=running --no-trunc --format \"{{json .}}\"";
+        private const string dockerInfoCommand = "info";
+        private const string dockerInfoArgs = "-f {{.Driver}}";
+        private const string dockerVersionCommand = "version";
+        private const string dockerVersionArgs = "-f {{.Server.Os}}";
+        private const string dockerInspectCommand = "inspect";
+        private const string dockerInspectArgs = "-f \"{{json .Platform}}\" ";
+        private static char[] charsToTrim = { ' ', '\"' };
+
+        public static bool LCOW(string hostname, out bool lcow)
```

Comment:
nit: Rename to TryGetLCOW and add comments above what it is doing.
"Querying the docker daemon to see if it supports Linux Containers on Windows (LCOW)"